Models/Http/UrlPlaceholderRenderInterface.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Infrastructure\Models\Http;
+
+
+class UrlPlaceholderRender
+{
+    public const GET_URL = 'getUrl';
+    public const GET_ONE_URL = 'getOneUrl';
+    public const CREATE_URL = 'createUrl';
+    public const UPDATE_URL = 'updateUrl';
+    public const DELETE_URL = 'deleteUrl';
+
+    /**
+     * @var array
+     */
+    private $urlsPlaceholders;
+
+    /**
+     * UrlPlaceholderRender constructor.
+     * @param array $urlsPlaceholders
+     */
+    public function __construct(array $urlsPlaceholders)
+    {
+        $this->urlsPlaceholders = $urlsPlaceholders;
+    }
+
+    /**
+     * @param array $data
+     * @return string
+     */
+    public function getUrl(array $data = []): string
+    {
+        return $this->render(self::GET_URL, $data);
+    }
+
+    /**
+     * @param array $data
+     * @return string
+     */
+    public function getOneUrl(array $data = []): string
```

Comment:
This method name is unclear for me.

This is not getter/setter and not command/query. Maybe you can think about other names.

Reply:
fixed